CHANGELOG.md

```diff
@@ -7,9 +7,13 @@ SPDX-License-Identifier: CC0-1.0
 
 ## Unreleased
 
+## [v1.1.0]
```

Comment:
Ah, I think the link for the release is missing too.

Reply:
I added it